client/src/components/FormField/RegisteredMFAMethodListField/RegisteredMFAMethodListField.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+import methodShape from '../../../types/registeredMethod';
+import ReadOnlyView from './ReadOnlyView';
+
+class RegisteredMFAMethodListField extends Component {
+  /**
+   * The backup and default methods are rendered separately
+   * @returns {*}
+   */
+  baseMethods() {
+    const { backupMethod, defaultMethod } = this.props;
+    let methods = this.props.registeredMethods;
```

Comment:
That results in different behaviour - the intention is to filter both the backup _and_ the default out if they're both present. Otherwise, I'd definitely have used early returns :)

I'll tweak it to use de-structuring.